gui/mozregui/log_report.py

```diff
@@ -1,7 +1,14 @@
-from PyQt4.QtCore import QObject, pyqtSlot as Slot, pyqtSignal as Signal
-from PyQt4.QtGui import QPlainTextEdit, QTextCursor
+from PyQt4.QtCore import QObject, pyqtSlot as Slot, pyqtSignal as Signal, Qt
+from PyQt4.QtGui import QPlainTextEdit, QTextCursor, QColor, \
+    QTextCharFormat, QPen
 from datetime import datetime
 
+color = {
+    'DEBUG': QColor(6, 146, 6),
+    'INFO': QColor(253, 248, 0),
+    'WARNING': QColor(255, 0, 0, 127),
```

Comment:
Please add a comment on each line to precise the name of the color you describe in QColor.
Like in report.py:
VERDICT_TO_ROW_COLORS = {
    "g": QColor(152, 251, 152),   # light green
    "b": QColor(250, 113, 113),  # light red
    "s": QColor(253, 248, 107),   # light yellow
    "r": QColor(225, 225, 225),   # light gray
}
It adds readability